tests/test_setup_utils_edge_cases.py

```
@@ -291,16 +295,32 @@ def test_install_service_create_file_failure(self):
+            ),
+            patch("mmrelay.setup_utils.logger"),
+            patch("builtins.input", return_value="y"),
         ):
-            with patch("mmrelay.setup_utils.create_service_file", return_value=False):
-                with patch("mmrelay.setup_utils.logger"):
-                    with patch(
-                        "builtins.input", return_value="y"
-                    ):  # Mock input to avoid stdin issues
-                        result = install_service()
-                        self.assertFalse(result)
+            mock_service_path.return_value = Path("/tmp/mmrelay-test.service")
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/mmrelay-test.service"

(S108)